src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -33,34 +28,23 @@ public LineResponse saveLine(LineRequest request) {
         Station upStation = stationService.findById(request.getUpStationId());
         Station downStation = stationService.findById(request.getDownStationId());
         Line persistLine = lineRepository.save(new Line(request.getName(), request.getColor(), upStation, downStation, request.getDistance()));
-        List<StationResponse> stations = getStations(persistLine).stream()
-                .map(it -> StationResponse.of(it))
-                .collect(Collectors.toList());
+
+        List<StationResponse> stations = StationResponse.of(persistLine.getStations());
         return LineResponse.of(persistLine, stations);
```

Comment:
```suggestion
        return LineResponse.of(persistLine);
```

와 같이 정적 팩토리로 역할을 넘겨줄 수 있겠네요 ㅎㅎ

Reply:
이 부분도 잠시 고민했었네요!!
우선 DTO의 역할이 너무 많아지는것기 아닌가? 라른 막연한 생각이였던 것 같아요
하지만 이미 DTO는 전송,변환 이렇게 두가지의 역할을 하고 있으니 DTO에서 변환을 해주는게 좋은것같네요!!